### PR TITLE
Added release notes templates and package indexes for Android and iOS.

### LINKED ISSUE #1485: Add package index for all languages
We currently have a package index for .NET, Java, Python, and JS/Typescript we need to add a package index for the others:

- [x] C
- [x] C++
- [x] Andriod
- [x] iOS
- [x] Go (Some info in https://github.com/Azure/azure-sdk/issues/1278).

### PR DESCRIPTION
**Pull request by vcolin7**
Fixes #1485.

Added new Relase Notes templates for both Android and iOS. Also added package indexes for both languages.

We will add details of this month's releases for both platforms once the templates are approved and merged.

### CODE REVIEW

File: releases/2020-10/ios.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Changelog
+
+Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
+
+### _Package name_
+
+- Major changes only!
+
+## Latest Releases
```
Comment (bsiegel) [inline releases/2020-10/ios.md:61]:
```suggestion
## Need help
```

File: releases/2020-10/android.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
+
+### _Package name_
+
+- Major changes only!
+  
+## Need help
+
+- For reference documentation visit the [Azure SDK for Android documentation](https://azure.github.io/azure-sdk-for-android/).
+- For tutorials, samples, quick starts and other documentation, visit the [Azure for Android repository](https://github.com/azure/azure-sdk-for-android/).
```
Comment (bsiegel) [inline releases/2020-10/android.md:62]:
```suggestion
- For tutorials, samples, quick starts and other documentation, visit the [Azure SDK for Android repository](https://github.com/azure/azure-sdk-for-android/).
```

File: releases/2020-10/ios.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+- For reference documentation visit the [Azure SDK for iOS documentation](https://azure.github.io/azure-sdk-for-ios/).
+- For tutorials, samples, quick starts and other documentation, visit [Azure for Mobile Developers](https://github.com/azure/azure-sdk-for-ios/).
```
Comment (bsiegel) [inline releases/2020-10/ios.md:64]:
```suggestion
- For tutorials, samples, quick starts and other documentation, visit the [Azure SDK for iOS repository](https://github.com/azure/azure-sdk-for-ios/).
```

File: releases/2020-10/ios.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-ios/issues/new/choose).
+- Check [previous questions](https://stackoverflow.com/questions/tagged/azure-ios-sdk) or ask new ones on
+ StackOverflow using the `azure-ios-sdk` tag.
```
Comment (bsiegel) [inline releases/2020-10/ios.md:67]:
```suggestion
 StackOverflow using the `azure` and `ios` tags.
```

File: releases/2020-10/ios.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+## Latest Releases
+
+View all the latest versions of iOS packages [here][android-latest-releases].
```
Comment (bsiegel) [inline releases/2020-10/ios.md:71]:
```suggestion
View all the latest versions of iOS packages [here][ios-latest-releases].
```

File: releases/2020-10/ios.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+- _Add packages_
+
+#### Beta
+
+- _Add packages_
+
+## Installation Instructions
+
+To install the latest GA and beta libraries, use the [Swift Package Manager](https://swift.org/package-manager/). If you are using a different tool, refer to its documentation on how to specify dependencies.
```
Comment (bsiegel) [inline releases/2020-10/ios.md:25]:
```suggestion
To install the latest GA and beta libraries, use the [Swift Package Manager](https://swift.org/package-manager/).
```

File: releases/2020-10/ios.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
+---
+title: Azure SDK for iOS (October 2020)
+layout: post
+tags: ios azure
+sidebar: releases_sidebar
+repository: azure/azure-sdk-for-ios
+---
+
+The Azure SDK team is pleased to make available the October 2020 client library release.
```
Comment (bsiegel) [inline releases/2020-10/ios.md:9]:
```suggestion
The Azure SDK team is pleased to make available the October 2020 client library releases.
```

**Review by bsiegel — DISMISSED** [review]

File: releases/2020-10/ios.md (hunk 6 above)
Comment (vcolin7) [reply releases/2020-10/ios.md:25]: @bsiegel I guess this means we expect people to only use Swift Package Manager, correct?

File: releases/2020-10/ios.md (hunk 4 above)
Comment (vcolin7) [reply releases/2020-10/ios.md:67]: Would it be a good idea to create a new tag for this in addition to those two?

File: releases/2020-10/ios.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+### _Package name_
+
+- Major changes only!
+
+## Latest Releases
+
+- For reference documentation visit the [Azure SDK for iOS documentation](https://azure.github.io/azure-sdk-for-ios/).
+- For tutorials, samples, quick starts and other documentation, visit [Azure for Mobile Developers](https://github.com/azure/azure-sdk-for-ios/).
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-ios/issues/new/choose).
+- Check [previous questions](https://stackoverflow.com/questions/tagged/azure-ios-sdk) or ask new ones on
```
Comment (vcolin7) [inline releases/2020-10/ios.md:66]: @bsiegel The comment below implies we won't use the tag specified in this link, so we might want to change it, unless we plan on using the new tag as well.

**Review by czubair — APPROVED** [review]

File: releases/2020-10/android.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+#### GA
+
+- _Add packages_
+
+#### Updates
+
+- _Add packages_
+
+#### Beta
```
Comment (anuchandy) [inline releases/2020-10/android.md:19]: our packages have beta suffix, but officially do we call it "beta" or "preview", java is putting under "preview" [header](https://github.com/Azure/azure-sdk/blob/master/releases/2020-09/java.md#preview), seems a typo in java-side.

**Review by anuchandy — APPROVED** [review]

File: releases/2020-10/android.md (hunk 9 above)
Comment (vcolin7) [reply releases/2020-10/android.md:19]: Thanks Anu, I will change this to preview on both templates.

Comment (vcolin7) [inline releases/2020-10/android.md:19]:
```suggestion
#### Preview
```

File: releases/2020-10/ios.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+#### GA
+
+- _Add packages_
+
+#### Updates
+
+- _Add packages_
+
+#### Beta
```
Comment (vcolin7) [inline releases/2020-10/ios.md:19]:
```suggestion
#### Preview
```

File: releases/2020-10/ios.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+
+- _Add packages_
+
+## Installation Instructions
+
+To install the latest GA and beta libraries, use the [Swift Package Manager](https://swift.org/package-manager/). If you are using a different tool, refer to its documentation on how to specify dependencies.
```
Comment (vcolin7) [inline releases/2020-10/ios.md:25]:
```suggestion
To install the latest GA and preview libraries, use the [Swift Package Manager](https://swift.org/package-manager/). If you are using a different tool, refer to its documentation on how to specify dependencies.
```

File: releases/2020-10/android.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+- _Add packages_
+
+## Installation Instructions
+
+To use the latest GA and beta libraries, refer to the dependency information below, which may be copied into your projects Gradle `build.gradle` or Maven `pom.xml` file as appropriate. If you are using a different build tool, refer to its documentation on how to specify dependencies.
```
Comment (vcolin7) [inline releases/2020-10/android.md:25]:
```suggestion
To use the latest GA and preview libraries, refer to the dependency information below, which may be copied into your projects Gradle `build.gradle` or Maven `pom.xml` file as appropriate. If you are using a different build tool, refer to its documentation on how to specify dependencies.
```

File: releases/2020-10/android.md (hunk 9 above)
Comment (vcolin7) [reply releases/2020-10/android.md:19]: It should be preview from what I recall, right @czubair?

File: releases/2020-10/ios.md (hunk 4 above)
Comment (bsiegel) [reply releases/2020-10/ios.md:67]: Probably, but this is what's in our README and is how users were told previously to tag questions about Colby's libraries. If we change this in the future we should do it in both places, and maybe see if we can migrate or back-tag those issues.

File: releases/2020-10/ios.md (hunk 6 above)
Comment (bsiegel) [reply releases/2020-10/ios.md]: SwiftPM is the only package manager that is supported by the SDK for iOS.

File: releases/2020-10/ios.md (hunk 8 above)
Comment (bsiegel) [reply releases/2020-10/ios.md:66]: Oh, yeah I didn't see that one. One sec.

File: releases/2020-10/ios.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+### _Package name_
+
+- Major changes only!
+
+## Need help
+
+- For reference documentation visit the [Azure SDK for iOS documentation](https://azure.github.io/azure-sdk-for-ios/).
+- For tutorials, samples, quick starts and other documentation, visit the [Azure SDK for iOS repository](https://github.com/azure/azure-sdk-for-ios/).
+- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-ios/issues/new/choose).
+- Check [previous questions](https://stackoverflow.com/questions/tagged/azure-ios-sdk) or ask new ones on
```
Comment (bsiegel) [inline releases/2020-10/ios.md:66]:
```suggestion
- Check [previous questions](https://stackoverflow.com/questions/tagged/azure+ios) or ask new ones on
```

File: releases/2020-10/android.md (hunk 9 above)
Comment (bsiegel) [reply releases/2020-10/android.md]: I believe we switched from saying "preview" to "beta" sometime around last month's release. Cala will know for sure.

Comment (vcolin7) [reply releases/2020-10/android.md]: Oh you are right, my bad.

File: _data/releases/latest/android-packages.csv [hunk 14]
```diff
@@ -0,0 +1,4 @@
+"Package","GroupId","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","Hide","Notes"
+"azure-communication-chat","com.azure.android","","1.0.0-beta.2","Communication Chat","Communication","communication","","https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/communication/azure-communication-chat","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/android-packages.csv:2]: You should set NA in the MSDocs and GHDocs field if the those links aren't valid.

Comment (weshaggard) [inline _data/releases/latest/android-packages.csv:2]:
```suggestion
"azure-communication-chat","com.azure.android","","1.0.0-beta.2","Communication Chat","Communication","communication","NA","NA","client","",""
```

File: _data/releases/latest/ios-packages.csv [hunk 15]
```diff
@@ -0,0 +1,4 @@
+"Package","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","Hide","Notes"
+"AzureCommunication","","1.0.0-beta.2","Communication Common","Communication","communication","","https://github.com/Azure/azure-sdk-for-ios/tree/master/sdk/communication/AzureCommunication","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/ios-packages.csv:2]:
```suggestion
"AzureCommunication","","1.0.0-beta.2","Communication Common","Communication","communication","NA","NA","client","",""
```

File: _includes/releases/pkgrow.md [hunk 16]
```diff
@@ -4,7 +4,7 @@
   <td>
     {% assign trimmedPackage = item.Package | remove: package_trim %}
 
-    {% assign package_url = package_url_template | replace: 'item.Package', item.Package | replace: 'item.TrimmedPackage', trimmedPackage | replace: 'item.GroupId', item.GroupId %}
+    {% assign package_url = package_url_template | replace: 'item.Package', item.Package | replace: 'item.TrimmedPackage', trimmedPackage | replace: 'item.GroupId', item.GroupId | | replace: 'item.RepoPath', item.RepoPath %}
```
Comment (weshaggard) [inline _includes/releases/pkgrow.md:7]:
```suggestion
    {% assign package_url = package_url_template | replace: 'item.Package', item.Package | replace: 'item.TrimmedPackage', trimmedPackage | replace: 'item.GroupId', item.GroupId | replace: 'item.RepoPath', item.RepoPath %}
```

File: docs/policies/releases.md [hunk 17]
```diff
@@ -270,6 +270,37 @@ An Embedded C release includes a Tag and Release (e.g. [1.0.0-preview.5](https:/
...
+
+**GA Release:** `1.1.0` -> `1.2.0-beta.1`
+
+**GA Hotfix Release:** `1.0.0` -> `1.0.1`
+
+#### iOS
+
+iOS releases the source code of the repository in a single unit of source code. It supports only the Swift Package Manager and does not ship packages to any package registry. Because the iOS repo ships from the `master` branch, code going into the `master` branch must be in a completed state and ready to ship.
+
+An iOS release includes a Tag and Release (e.g. [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-ios/releases/tag/1.0.0-preview.2)) on GitHub and documentation as GitHub Pages (e.g. [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-ios/tree/1.0.0-beta.2)).
```
Comment (weshaggard) [inline docs/policies/releases.md:294]:
```suggestion
An iOS release includes a Tag and Release (e.g. [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-ios/releases/tag/1.0.0-beta.2)) on GitHub and documentation as GitHub Pages (e.g. [1.0.0-beta.2](https://github.com/Azure/azure-sdk-for-ios/tree/1.0.0-beta.2)).
```

File: eng/scripts/Update-Release-Versions.ps1 [hunk 18]
```diff
@@ -360,6 +360,102 @@ function Update-cpp-Packages($packageList)
...
+  $valid = $valid -and (CheckLink ("https://github.com/Azure/azure-sdk-for-android/tree/{0}_{1}/sdk/{2}/{0}/" -f $pkg.Package, $version, $pkg.RepoPath))
+  $valid = $valid -and (CheckLink ("https://search.maven.org/artifact/{2}/{0}/{1}/aar/" -f $pkg.Package, $version, $pkg.GroupId))
+  return $valid;
+}
+
+function Update-android-Packages($packageList)
+{
+  foreach ($pkg in $packageList)
+  {
+    $version = GetVersionWebContent "android" $pkg.Package "latest-ga"
```
Comment (weshaggard) [inline eng/scripts/Update-Release-Versions.ps1:375]: This is only ever going to work if we are publishing github.io docs and I don't think we are at least for now. This means that we will have to update versions manually until that work is done.

File: eng/scripts/Update-Release-Versions.ps1 [hunk 19]
```diff
@@ -386,6 +482,8 @@ switch($language)
     Output-Latest-Versions "python"
     Output-Latest-Versions "c"
     Output-Latest-Versions "cpp"
+    Output-Latest-Versions "android"
```
Comment (weshaggard) [inline eng/scripts/Update-Release-Versions.ps1:485]: Can you comment out these 2 lines until the github.io doc publishing is done? We can keep the other code for now and we can enable uncomment these to enable it later once that other work is done.

File: _data/releases/latest/ios-packages.csv (hunk 15 above)
Comment (weshaggard) [reply _data/releases/latest/ios-packages.csv]: Similar update for all the packages.

**Comment by vcolin7** [comment]
FINALLY!

File: docs/policies/releases.md [hunk 20]
```diff
@@ -270,6 +270,37 @@ An Embedded C release includes a Tag and Release (e.g. [1.0.0-preview.5](https:/
...
 **GA Hotfix Release:** `1.0.0` -> `1.0.1`
 
+#### Android
+
+Maven supports the [convention](https://cwiki.apache.org/confluence/display/MAVENOLD/Versioning) `MAJOR.MINOR.PATCH-QUALIFIER`, which doesn't support SemVer 2 sorting for pre-releases so we have to use a special convention based on their [versioning code](https://github.com/apache/maven/blob/master/maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java). The preferred format for version numbers is:
+
+- `X.Y.Z-alpha.YYYYMMDD.r` (`r` is based on the number of builds performed on the given day) for daily alpha releases.
+- `X.Y.Z-beta.N` for beta releases.
+
+Beta packages are published directly to the Maven central registry. Alpha packages will be published to the isolated [azure-sdk-for-android](https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-android) DevOps feed. To consume an alpha package you should either pin to a specific version or use a version specifier like `(X.Y.Z-alpha, X.Y.Z-beta)` to get the latest alpha release while avoiding any potential beta versions which might contain older changes.
```
Comment (weshaggard) [inline docs/policies/releases.md:280]: FYI I just created this feed https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-android. We will need make sure there is a work-item setup to publish to this feed.

**Review by weshaggard — APPROVED** [review]

File: docs/policies/releases.md (hunk 20 above)
Comment (vcolin7) [reply docs/policies/releases.md:280]: @weshaggard Just curious, do we have one for iOS already set up as well?

Comment (weshaggard) [reply docs/policies/releases.md:280]: iOS doesn't use a devops feed it only uses the source repo.

**Pull request merged**